backend/zeno_backend/server.py

```diff
@@ -43,6 +44,8 @@ def get_server() -> FastAPI:
     Returns:
         FastAPI: FastAPI endpoint
     """
+    _ = db_pool.open()
+
```

Comment:
1. doesn't this need to be awaited?
2. why do we need the `_ =`
3. Should we move this to a function in the DB file instead that is called when getting the pool?

Reply:
Done!